sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusSessionProcessorOptions.cs

```diff
@@ -44,7 +44,7 @@ public int PrefetchCount
 
         /// <summary>
         /// Gets or sets the maximum duration within which the session lock will be renewed automatically. This value
-        /// should be greater than the queue's LockDuration Property.
+        /// should be greater than the queue's LockDuration Property. The default value is 5 minutes.
```

Comment:
```suggestion
        /// should be greater than the queue's LockDuration Property. The default value is 5 minutes.
```

The default text would be a good candidate to move to a sibling `<value>` tag for better visibility in the docs.